src/components/card.js

```diff
@@ -11,7 +11,7 @@ const sizes = Object.freeze({
 })
 
 const Card = ({ classes, size, ...rest }) => {
-  const _classes = { ...classes, root: `card__root-container ${['rounded-sm', sizes[size], classes.root].join(' ')}` }
+  const _classes = { ...classes, root: `card__root-container ${[classes.root, 'rounded-sm', sizes[size]].join(' ')}` }
```

Comment:
Ok, so if we construct a class for all components  in the card-base, I would add the empty option string for classes here, so instead of 
```
classes.root
```
we would have:
```
classes.root || ''
```

and delete the option in the card-base as we will always pass a class for each component and it won't be undefined.

Reply:
> and delete the option in the card-base as we will always pass a class for each component and it won't be undefined.

👌🏻 I'll update the classes prop from PropTypes.object to PropTypes.object.isRequired to make it required.

Reply:
> The join() method will convert any undefined or null elements to an empty string, rather than the strings "null" or "undefined". ([ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join))

Oh, did not notice we work here with a list & join... all good!